docs/user-guide/home/global-modules/README.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_navigation:
+  title: Global modules
+  priority: 999
+description: Global modules in OpenProject
+keywords: global modules, project overarching modules, global index pages
+---
+# Global modules
+
+Global modules present an overview of the projects that you are a member of or have permissions to see. Here you will find a summary of all the entries in the respective modules across your projects, such as **Activity, Work packages, Calendars, Boards**, etc.
+
+To find **Global modules** menu simply click on the logo in the header of the application, it will be displayed on the left side. 
```

Comment:
@MayaBerd maybe we should add also that you can access the global modules from the top right grid menu icon.

Reply:
Excellent suggestion, I added it. Thank you.